llvm/docs/DirectX/DXILResources.rst

```diff
@@ -128,6 +129,8 @@ There are a few fields to describe variants of all of these types:
        writeable) and UAVs (writeable).
    * - IsROV
      - Whether the UAV is a rasterizer ordered view. Always ``0`` for SRVs.
+   * - IsSigned
+     - Whether the element type is signed ("dx.TypedBuffer" only)
```

Comment:
What would it mean to have an unsigned float?  I don't quite understand how this isn't inferable from the element type.  Wonder if some more explanation in this doc might be warranted?

Reply:
Added a couple of sentences to make this clearer